types/fixture/testutil.go

```diff
@@ -123,11 +130,14 @@ func (f fixture) Signers() []sdk.AccAddress {
 }
 
 func (f fixture) InitGenesis(ctx sdk.Context, genesisData map[string]json.RawMessage) (abci.ResponseInitChain, error) {
-	return initGenesis(ctx, f.cdc, genesisData, []abci.ValidatorUpdate{}, f.initGenesisHandlers)
+	// we inject the mock module genesis with bogus data in order to bypass the check for validator updates.
+	// since the testing fixture doesn't require validators/validator updates, the check fails otherwise.
+	genesisData[MockModule{}.Name()] = []byte(`{"foo":"bar"}`)
```

Comment:
We don't need to provide false values. This can simply be an empty struct.
```suggestion
	// we inject the mock module genesis in order to bypass the check for validator updates.
	// since the testing fixture doesn't require validators/validator updates, the check fails otherwise.
	genesisData[MockModule{}.Name()] = []byte(`{}`)
```